packages/blade/src/components/Amount/Amount.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title } from '@storybook/addon-docs';
+import type { AmountProps } from './Amount';
+import { Amount as AmountComponent } from './Amount';
+import BaseBox from '~components/Box/BaseBox';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+import { Text } from '~components/Typography';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=WfWp7qiwZ3lvvbdw-0',
+        bankingTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=WfWp7qiwZ3lvvbdw-0',
+      }}
+      componentName="Amount"
+      componentDescription="Amounts are used to show small amount of color coded metadata, which are ideal for getting user attention."
+    >
+      <Title>Usage</Title>
+      <Sandbox>
+        {`
+        import { Amount } from '@razorpay/blade/components';
+        
+        function App(): JSX.Element {
+          const value = 21222.2;
+          return (
+            <Amount intent="neutral" value={value} />  
+          )
+        }
+
+        export default App;
+        `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Amount',
+  component: AmountComponent,
+  argTypes: {},
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<AmountProps>;
+
+const AmountDefaultTemplate: ComponentStory<typeof AmountComponent> = (args) => {
+  return (
+    <BaseBox
+      display="flex"
+      alignItems="baseline"
+      justifyContent="center"
+      paddingRight="spacing.3"
+      paddingTop="spacing.2"
+    >
+      <AmountComponent {...args} />
+    </BaseBox>
+  );
+};
+
+export const Amount = AmountDefaultTemplate.bind({});
+Amount.args = {
+  value: 122345.678,
+  intent: 'neutral',
+  size: 'title-medium',
+  suffix: 'decimals',
+  isAffixSubtle: true,
+};
+Amount.storyName = 'Default';
+
+const AmountSizesTemplate: ComponentStory<typeof AmountComponent> = ({ ...args }) => {
```

Comment:
There was a minor issue here, but the design has these conditions for prefix to be bold 
- if isAffixSubtle is true, then bold
- if contains bold then, then bold. 
In anyother case it will be regular.

Reply:
This is resolved now? bold and regular works as it should, right?